client/my-sites/customer-home/main.jsx

```diff
@@ -197,6 +218,53 @@ const Home = ( {
 		return null;
 	};
 
+	const renderDnsSettingsDiagnosticNotice = () => {
+		if (
+			dismissedEmailDnsDiagnostics ||
+			isFetchingDomainDiagnostics ||
+			! emailDnsDiagnostics ||
+			emailDnsDiagnostics.code === 'domain_not_mapped_to_atomic_site' ||
+			emailDnsDiagnostics.all_essential_email_dns_records_are_correct
+		) {
+			return null;
+		}
+		const hasErrors = Object.values( emailDnsDiagnostics.records ).some(
+			( record ) => record.status === 'incorrect'
+		);
+
+		if ( hasErrors ) {
```

Comment:
I think we don't actually need this check because if `all_essential_email_dns_records_are_correct` is false, this means there is some issue with the domain and we should show the notice 🤔